imap_processing/codice/codice_l1a.py

```diff
@@ -97,12 +97,12 @@ def create_science_dataset(self, start_time: np.datetime64) -> xr.Dataset:
         cdf_attrs.add_instrument_global_attrs("codice")
         cdf_attrs.add_instrument_variable_attrs("codice", "l1a")
 
-        # Define DataArrays for the coordinates
+        # Define coordinates
         epoch = xr.DataArray(
-            [start_time],
+            [start_time, start_time],  # TODO: Fix after SIT-3 (see note below)
```

Comment:
Can you make these different values too? The upcoming cdflib will complain if they are duplicates.
`[start_time, start_time + np.timedelta64(1, "s")]`

Reply:
Ah yes, great catch!